README.md

```diff
@@ -20,66 +20,113 @@ A collection of Schemata inspired by io-ts-types and validators.js.
 
 </div>
 
-## Table of Contents
+## Introduction
 
-<!-- AUTO-GENERATED-CONTENT:START (TOC) -->
+A schema is an expression of a type structure that can be used to generate typeclass instances from a single declaration. The following example constructs a `User` schema from which the underlying domain type can be extracted. `schemata-ts` generates the following typeclasses: `Arbitrary`, `Decoder`, `Encoder`, `Guard`, `Eq`, `TaskDecoder`, and `Type`.
 
-- [Disclaimer](#disclaimer)
```

Comment:
Removing TOC because I learned that Github has one built in

Reply:
Oh neat, that's new!